terraform/custom_domains/environment_domains/workspace_variables/register_qa.tfvars.json

```diff
@@ -0,0 +1,54 @@
+{
+  "multiple_hosted_zones": true,
+  "enable_alerting": true,
+  "pg_actiongroup_name": "s189p01-att-production-ag",
+  "pg_actiongroup_rg": "s189p01-att-pd-rg",
```

Comment:
We should think about getting an action group setup for `rtt`

Reply:
should this be for each environment or we feed off production ?